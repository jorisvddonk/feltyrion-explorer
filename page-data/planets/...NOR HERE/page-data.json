{"componentChunkName":"component---src-templates-planet-info-tsx","path":"/planets/...NOR HERE","result":{"data":{"planet":{"id":"planet_...NOR HERE...","data":{"name":"...NOR HERE...","index":"10","x":4173200,"y":-5584233,"z":-640733},"childrenGuideEntry":[{"data":{"text":" A VERY ACTIVE VOLCANIC WORLD. INTERNALLY HOT, SURFACE TO"}},{"data":{"text":"UNSTABLE FOR LANDING. (STARGAZER. EPOC 6011; 585.146)"}}]}},"pageContext":{"slug":"/planets/...NOR HERE"}}}