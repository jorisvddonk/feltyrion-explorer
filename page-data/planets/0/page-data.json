{"componentChunkName":"component---src-templates-planet-info-tsx","path":"/planets/0","result":{"data":{"planet":{"id":"planet_0","data":{"name":"0","index":"12","x":4173200,"y":-5584233,"z":-640733},"childrenGuideEntry":[{"data":{"text":" NOTHING HERE EITHER. HOW MANY OF THESE MOONS ARE THERE?"}},{"data":{"text":"(STARGAZER. EPOC 6011; 585.146)"}}]}},"pageContext":{"slug":"/planets/0"}}}