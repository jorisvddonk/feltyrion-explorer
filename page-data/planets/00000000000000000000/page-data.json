{"componentChunkName":"component---src-templates-planet-info-tsx","path":"/planets/00000000000000000000","result":{"data":{"planet":{"id":"planet_00000000000000000000","data":{"name":"00000000000000000000","index":"15","x":574288,"y":-535718,"z":-933464},"childrenGuideEntry":[]}},"pageContext":{"slug":"/planets/00000000000000000000"}}}