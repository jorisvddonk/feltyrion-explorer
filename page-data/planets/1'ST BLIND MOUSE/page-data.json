{"componentChunkName":"component---src-templates-planet-info-tsx","path":"/planets/1'ST BLIND MOUSE","result":{"data":{"planet":{"id":"planet_1'ST BLIND MOUSE","data":{"name":"1'ST BLIND MOUSE","index":"01","x":3183920,"y":-5560454,"z":-1209816},"childrenGuideEntry":[{"data":{"text":" DISCOVERED BY MRUQE; EPOC 6011-568-471"}},{"data":{"text":"HAS A GREAT BELT AROUND IT."}}]}},"pageContext":{"slug":"/planets/1'ST BLIND MOUSE"}}}