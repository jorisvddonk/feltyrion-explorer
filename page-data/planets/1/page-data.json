{"componentChunkName":"component---src-templates-planet-info-tsx","path":"/planets/1","result":{"data":{"planet":{"id":"planet_1","data":{"name":"1","index":"13","x":-643216,"y":-9578380,"z":958538},"childrenGuideEntry":[]}},"pageContext":{"slug":"/planets/1"}}}