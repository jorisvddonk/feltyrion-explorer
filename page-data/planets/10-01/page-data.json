{"componentChunkName":"component---src-templates-planet-info-tsx","path":"/planets/10-01","result":{"data":{"planet":{"id":"planet_10-01","data":{"name":"10-01","index":"01","x":-737520,"y":-5031334,"z":5347062},"childrenGuideEntry":[]}},"pageContext":{"slug":"/planets/10-01"}}}