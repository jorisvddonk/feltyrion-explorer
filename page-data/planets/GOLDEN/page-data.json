{"componentChunkName":"component---src-templates-planet-info-tsx","path":"/planets/GOLDEN","result":{"data":{"planet":{"id":"planet_GOLDEN","data":{"name":"GOLDEN","index":"01","x":3666416,"y":-4731653,"z":-785554},"childrenGuideEntry":[{"data":{"text":"FURTHER NOTES: LARGE, NOT CONSISTENT, COVERED WITH DENSE CLOUDS. NOT"}},{"data":{"text":"VERY REFLECTIVE, EVEN AT THIS DISTANCE FROM THE STAR. MANY LIGHTNING"}},{"data":{"text":"STORMS. SKINNYMON EPOC 6011 574.945"}}]}},"pageContext":{"slug":"/planets/GOLDEN"}}}